includes/wcpdf-functions.php

```diff
@@ -350,6 +355,38 @@ function wcpdf_output_error( $message, $level = 'error', $e = null ) {
 	<?php
 }
 
+/**
+ * Error handling function
+ *
+ * @param string $message
+ * @param string $error_handling
+ * @param bool   $log_error
+ * @param string $log_level
+ * @return mixed
+ */
+function wcpdf_error_handling( $message, $error_handling = 'exception', $log_error = false, $log_level = 'error' ) {
+	if ( $log_error ) {
+		wcpdf_log_error( $message, $log_level );
+	}
+	
+	switch ( $error_handling ) {
+		case 'exception':
+			throw new \Exception( $message );
+			break;
+		case 'output':
+			wcpdf_output_error( $message, $log_level );
+			break;
+		case 'return':
+			return $message;
+			break;
+		case 'false':
+			return false;
+			break;
```

Comment:
I think it can be removed since it is the same as the return at the end of the function.